### ISSUE TITLE
Final copy/design issues to fix discovered in Phase 2 Review

### ISSUE THREAD

**Issue by ALRubinger** [TBD54566975/developer.tbd.website#392]
Issues spotted in the comments, please! We can put all requests in here to be knocked off in design/tech/copy QA Friday the 14th of April before launch.

Based on the deploy preview branch `phase-2`: https://deploy-preview-365--tbd-website-developer.netlify.app/docs/

Not all of these are required to be fixed if they're more involved than a quick pass. In those cases we'll carve them off into new issues.

TODO - inventory these and create new issues as part of scoping Phase 3.

**Comment by ALRubinger** [comment]
Some more right margin on the header: 
<img width="170" alt="image" src="https://user-images.githubusercontent.com/199891/231912081-7a4abe75-5ba5-419a-a0a8-73a7d17922ad.png">

**Comment by ALRubinger** [comment]
Align buttons on Docs Home (Getting Started):
<img width="923" alt="image" src="https://user-images.githubusercontent.com/199891/231912177-d325c955-cb88-41a9-84f6-61507f135970.png">

**Comment by ALRubinger** [comment]
Sidebar font weight/face/size consistency? Something @dayhaysoos and I both identified seems a little off and could probably be tightened up a bit:
<img width="285" alt="image" src="https://user-images.githubusercontent.com/199891/231912262-e2e8176a-2a9d-4427-85e8-331d50c470cd.png">

**Comment by ALRubinger** [comment]
`docs/web5/learn/decentralized-identifiers`: Didn't @bobbilee19 have new images for this?

<img width="739" alt="image" src="https://user-images.githubusercontent.com/199891/231912369-5574c240-713c-4c0c-a6d6-7975486a24c8.png">

#382

**Comment by ALRubinger** [comment]
Content section of the docs pages, for instance the Quickstart, has no right margin:
<img width="1224" alt="image" src="https://user-images.githubusercontent.com/199891/231915283-09dd23c6-a85b-47f3-8b8f-1b101f583d43.png">

**Comment by ALRubinger** [comment]
Code blocks in Quickstart should use monospaced font:

<img width="330" alt="image" src="https://user-images.githubusercontent.com/199891/231917274-4656b77c-8f5b-4188-b1ce-4525cd27575c.png">

**Comment by ALRubinger** [comment]
Code blocks in the collapsable "details" section of the Quickstart are "greyed out":
<img width="815" alt="image" src="https://user-images.githubusercontent.com/199891/231917602-801841d4-b2a2-471b-b6fb-377d4290a6cb.png">

**Comment by ALRubinger** [comment]
![Uploading 7327E246-2337-430F-BAB5-6AD8B065647B.jpeg…]()

Padding on mobile as requested by @angiejones in Discord

**Comment by angiejones** [comment]
Change "Without NPM" to be "With CDN" and remove the title within the collapsed section

Currently
<img width="295" alt="image" src="https://user-images.githubusercontent.com/15972783/231948505-fb3875e8-0920-40c0-9e3c-580ccc56d1c2.png">

Proposed
<img width="173" alt="image" src="https://user-images.githubusercontent.com/15972783/231948645-06bc2bed-f294-4652-b275-cdfaa3b17e92.png">

The rationale is that it gets confusing on step 1 since the import is done within an IDE and not a npm command

**Comment by angiejones** [comment]
Quickstart page:

Add one sentence blurb about DIDs and link to DID Learn guide

<img width="1181" alt="image" src="https://user-images.githubusercontent.com/15972783/231949700-71f99056-038a-4cc9-b63e-d134be6c05ec.png">

**Comment by angiejones** [comment]
What's the call here? Remove the note and if the signature hasnt changed, add a sentence saying ion is the DID method. Link the words 'DID method' to the Glossary page

<img width="1195" alt="image" src="https://user-images.githubusercontent.com/15972783/231950028-dee2aec9-a925-40f9-bc10-9cee9117cdab.png">

**Comment by angiejones** [comment]
Add placeholder text here (such as "Click Run to see how to create a DID" instead of `...`. It took me a second to figure out what that was and why that was there

<img width="339" alt="image" src="https://user-images.githubusercontent.com/15972783/231950229-3a3fa586-548d-4d79-9294-fc0b5ddde98a.png">

**Comment by angiejones** [comment]
🚨 This is incorrect. That's not what `register` does. (See @frankhinek I told you this was confusing!). Team, this doc edit is a MUST-CHANGE! We cannot go live with this description. The correct description of `register` can be found on the Create a [DID guide](https://deploy-preview-365--tbd-website-developer.netlify.app/docs/web5/build/decentralized-identifiers/how-to-create-did#register-did-keys-in-decentralized-web-node)

<img width="1158" alt="image" src="https://user-images.githubusercontent.com/15972783/231951008-14add681-916b-489a-8c6c-f6033ba89432.png">

**Comment by angiejones** [comment]
Remove first `console.log` here
<img width="548" alt="image" src="https://user-images.githubusercontent.com/15972783/231951908-1e70cb0c-b03e-4b01-85ec-431978a4ddb5.png">

**Comment by angiejones** [comment]
missing the second end `:` for the `::note::` so the note isn't showing

<img width="760" alt="image" src="https://user-images.githubusercontent.com/15972783/231952485-8f516593-e8db-47ed-83d0-72e6f59b3a24.png">

**Comment by angiejones** [comment]
### Quickstart

Change `you'ved saved` to `that was saved`

We want want to confuse them as though this only returns things they've written personally, and not all data that matches the query...regardless of who wrote it there.

<img width="760" alt="image" src="https://user-images.githubusercontent.com/15972783/231952924-879f33d1-a5bb-4324-adfe-c2e3e41790ac.png">

**Comment by angiejones** [comment]
### Quickstart

Change `Found 1 entries!` to `Found 1 entry`
<img width="679" alt="image" src="https://user-images.githubusercontent.com/15972783/231953371-54522c9e-d07e-4621-ad7c-8a781dd8446b.png">

**Comment by angiejones** [comment]
Remove `console.log`. It made me think that's what would show up when I clicked Run
<img width="766" alt="image" src="https://user-images.githubusercontent.com/15972783/231954643-05465c20-b23b-4925-b4b5-13983dfecf20.png">

**Comment by angiejones** [comment]
### Sidebar

"Delete Data from a Decentralized Web Node" should be the last guide under Build->Decentralized Identifiers

![image](https://user-images.githubusercontent.com/15972783/231954820-66381e31-1613-4eaf-b188-bb506ea9d4b5.png)

**Comment by angiejones** [comment]
### DWN Guides

@chris-tbd can you update each guide to give one sentence explaining what's returned after executing the code snippet. Especially since we don't have a API reference. I don't know what I'm expecting back.

**Comment by angiejones** [comment]
### Quickstart

1. Remove `console.log` statements
2. Add a sentence or two after the heading
3. Explain what the last line does
4. Edit the sentence before the Run button

<img width="974" alt="image" src="https://user-images.githubusercontent.com/15972783/231955638-bb58416c-f417-4239-bb52-8bddb1cf1a60.png">

**Comment by angiejones** [comment]
### Quickstart, #7 

I'm not sure what the method returns exactly. Can we explain?

**Comment by angiejones** [comment]
### Quickstart

1. Remove `console.log` line
2. Indent argument list

<img width="708" alt="image" src="https://user-images.githubusercontent.com/15972783/231959378-3ed23730-db74-4d89-be59-39d9d523c09e.png">

**Comment by angiejones** [comment]
### Quickstart

Make Next Steps a header and add links

<img width="785" alt="image" src="https://user-images.githubusercontent.com/15972783/231959859-eefac2c5-40f9-4a89-a882-d6194b96d744.png">

**Comment by angiejones** [comment]
### Quickstart

Change `Deleted 1 entries` to `Deleted 1 entry`

<img width="273" alt="image" src="https://user-images.githubusercontent.com/15972783/231960228-29ffd22b-ae09-4c78-8d38-06f37511eeb5.png">

**Comment by angiejones** [comment]
### Quickstart

Add a description under header. Is it deleted forever? Can they restore it? Can anyone delete from the DWN?

<img width="502" alt="image" src="https://user-images.githubusercontent.com/15972783/231960456-e95e573a-d6e8-4d7c-b32c-ccc1065ba0c7.png">

**Comment by angiejones** [comment]
### Quickstart

There needs to be a closing. I got to the end of the Quickstart and from the mind of a newcomer, I have no idea what I've done here or what else I could do. I don't get the purpose. I stored an image where exactly? Where is my DWN? Does it persist once I leave this page? How can others read and write from this DWN I just created if I give them the DID that was assigned to me while doing this?

Basically take a step back and think what questions you'd have if you knew nothing about Web5 and did a Quickstart to learn how it works. Our closing (and in some of the steps above it) should answer those questions before they have to be asked.

**Comment by angiejones** [comment]
### Quickstart

Step 5's font is smaller than all the other steps

<img width="462" alt="image" src="https://user-images.githubusercontent.com/15972783/231962057-9ef8a89b-8051-47e8-af3e-dd41fe47b5bb.png">

**Comment by angiejones** [comment]
### Quickstart

Are the steps written as headings? How come they aren't showing up as right menu items?

**Comment by angiejones** [comment]
### Build a ToDo App

Add `import` line to [Loading or Creating User Data](https://deploy-preview-365--tbd-website-developer.netlify.app/docs/web5/build/apps/todo-app-tutorial#loading-or-creating-user-data)

cc @chris-tbd

**Comment by angiejones** [comment]
### Build a ToDo App

In the [Loading or Creating User Data](https://deploy-preview-365--tbd-website-developer.netlify.app/docs/web5/build/apps/todo-app-tutorial/#loading-or-creating-user-data) section, the following statement explains the code in the reverse order. While reading `if the data is available...`, I'd expect that to match the `if` statement, but it's reverse.

> If the data is available we can use that to access their DWN via Web5.js, otherwise we create a new did for them and a DWN to store their data.

```
 if(!localStorage.getItem('dwn-info')){
        // User does not have an "account," so create one for them
    const myDid = await web5.did.create('ion');
    registerInfo = {
      connected : true,
      did       : myDid.id,
      endpoint  : 'app://dwn', 
      keys      : myDid.keys[0].keypair
    };
    localStorage.setItem('dwn-info', JSON.stringify(registerInfo));
  } else {
        // User has an "account," so load their data
    registerInfo = JSON.parse(localStorage.getItem('dwn-info'));
  }
```

**Comment by ALRubinger** [comment]
> ### Quickstart
> What's the call here? Remove the note and if the signature hasnt changed, add a sentence saying ion is the DID method. Link the words 'DID method' to the Glossary page
> 
> <img alt="image" width="1195" src="https://user-images.githubusercontent.com/15972783/231950028-dee2aec9-a925-40f9-bc10-9cee9117cdab.png">

Removed the note. Opened for us to address in another Phase as #395.

**Comment by ALRubinger** [comment]
> ### Quickstart
> Add one sentence blurb about DIDs and link to DID Learn guide
> 
> <img alt="image" width="1181" src="https://user-images.githubusercontent.com/15972783/231949700-71f99056-038a-4cc9-b63e-d134be6c05ec.png">

<img width="833" alt="image" src="https://user-images.githubusercontent.com/199891/232030648-7fa285a9-51fd-48c5-b004-292723f703ff.png">

**Comment by ALRubinger** [comment]
> Code blocks in the collapsable "details" section of the Quickstart are "greyed out": <img alt="image" width="815" src="https://user-images.githubusercontent.com/199891/231917602-801841d4-b2a2-471b-b6fb-377d4290a6cb.png">

Fixing this by not having the code blocks wrapped in a `<p>` tag

**Comment by ALRubinger** [comment]
> ### Quickstart
> Add placeholder text here (such as "Click Run to see result") instead of `...`. It took me a second to figure out what that was and why that was there
> 
> <img alt="image" width="339" src="https://user-images.githubusercontent.com/15972783/231950229-3a3fa586-548d-4d79-9294-fc0b5ddde98a.png">

<img width="305" alt="image" src="https://user-images.githubusercontent.com/199891/232032211-4269915d-5a7f-446e-80f2-2129dc27a487.png">

**Comment by ALRubinger** [comment]
> 🚨 This is incorrect. That's not what register does. (See @frankhinek I told you this was confusing!). Team, this doc edit is a MUST-CHANGE! We cannot go live with this description. The correct description of register can be found on the [Create a DID guide](https://deploy-preview-365--tbd-website-developer.netlify.app/docs/web5/build/decentralized-identifiers/how-to-create-did#register-did-keys-in-decentralized-web-node)

@angiejones I'm not sure that's true. The text about what "register" does is there is based on a convo I had with @csuwildcat and I had him visually approve that text for accuracy.

Sounds like we need review from OSE team on what "registering a DID with Web5" does. And this should be in the forthcoming JSDocs for Web5 on the "register" method, which would clear this up too.

Def agree with you that yes this is confusing. :D

**Comment by csuwildcat** [comment]
Just FYI: it's now Web5.did.manager.set(), and I think management is a good/proper way to describe what it's doing.

**Comment by angiejones** [comment]
> Just FYI: it's now Web5.did.manager.set(), and I think management is a good/proper way to describe what it's doing.

@chris-tbd can you update the tutorial and Create a DID guide to use the new function name?

**Comment by EbonyLouis** [comment]
> ### Quickstart
> Add placeholder text here (such as "Click Run to see result") instead of `...`. It took me a second to figure out what that was and why that was there
> 
> <img alt="image" width="339" src="https://user-images.githubusercontent.com/15972783/231950229-3a3fa586-548d-4d79-9294-fc0b5ddde98a.png">

With this there should be consistency , so step 3 it now says " click run etc" but steps 4-8 either have the "..." still or nothing, can we hide the response until Run is clicked like in step 4 and maybe just have every button say "Click to Run" so its consistent?

**Comment by dayhaysoos** [comment]
@angiejones  regarding quick start step 7, it's supposed to show the image that was uploaded. That's straight up not happening, will take a closer look later

**Comment by dayhaysoos** [comment]
I agree with @EbonyLouis on "Click to Run". I think the `...` should at least be labelled with "Results" or something. Looking further into it now

**Comment by angiejones** [comment]
> second

@ALRubinger a few more places where placeholder is needed instead of `...`

**Comment by csuwildcat** [comment]
Possible prose to add for explaining the interaction model of DWNs more: "DWeb Nodes are personal datastores that live on your devices and external places you choose. When you write an app that uses Web5 SDK to write data to a user's DWeb Node it is committed locally, and replicated to any other DWeb Nodes connected to the user's DID"

**Comment by chris-tbd** [comment]
> ### DWN Guides
> @chris-tbd can you update each guide to give one sentence explaining what's returned after executing the code snippet. Especially since we don't have a API reference. I don't know what I'm expecting back.

@csuwildcat could you send me the return object here? I don't see it documented anywhere

**Comment by angiejones** [comment]
> Possible prose to add for explaining the interaction model of DWNs more: "DWeb Nodes are personal datastores that live on your devices and external places you choose. When you write an app that uses Web5 SDK to write data to a user's DWeb Node it is committed locally, and replicated to any other DWeb Nodes connected to the user's DID"

thank you! cc @bobbilee19 @EbonyLouis

**Comment by ALRubinger** [comment]
> ### Quickstart
> 🚨 There needs to be a closing. I got to the end of the Quickstart and from the mind of a newcomer, I have no idea what I've done here or what else I could do. I don't get the purpose. I stored an image where exactly? Where is my DWN? How is a DWN different from localStorage? Does it persist once I leave this page? How can others read and write from this DWN I just created if I give them the DID that was assigned to me while doing this?
> 
> Basically take a step back and think what questions you'd have if you knew nothing about Web5 and did a Quickstart to learn how it works. Our closing (and in some of the steps above it) should answer those questions before they have to be asked.

I broke this out into blocker #401 - team is on it!

**Comment by ALRubinger** [comment]
> > second
> 
> @ALRubinger a few more places where placeholder is needed instead of `...`

Agreed and I wanna handle after `phase-2`; it involves updating both in Quickstart and in `web5-quickstart-widgets`.

**Comment by ALRubinger** [comment]
> I agree with @EbonyLouis on "Click to Run". I think the `...` should at least be labelled with "Results" or something. Looking further into it now

@dayhaysoos agree and per comment above, let's do after `phase-2` because we should probably sync changes where this is embedded on the Quickstart w/ the `web5-quickstart-widgets` repo where this is runnable standalone.

**Comment by chris-tbd** [comment]
@angiejones fixed the call to `register` and added the protocol page here https://github.com/TBD54566975/developer.tbd.website/pull/400

**Comment by ALRubinger** [comment]
> > Just FYI: it's now Web5.did.manager.set(), and I think management is a good/proper way to describe what it's doing.
> 
> @chris-tbd can you update the tutorial and Create a DID guide to use the new function name?

The runnable code at https://github.com/TBD54566975/web5-quickstart-widgets is the authoritative source for the code; it's what proves it's working. With a new web5-js version in there, any code changes need to go there to validate what we surface in docs.

**Comment by ALRubinger** [comment]
Taking design off the critical path for Phase 3
